backend/app/api/ws/chat_namespace.py

```
@@ -633,9 +626,7 @@ async def on_chat_send(self, sid: str, data: dict) -> dict:
 
             # Create task and subtasks using unified function
             # Automatically handles Chat Shell vs Executor path based on team configuration
-            logger.info(
-                f"[WS] chat:send calling create_chat_task (supports_direct_chat={supports_direct_chat})..."
-            )
+            logger.info(f"[WS] chat:send calling create_chat_task...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)